components/org.wso2.carbon.identity.application.authenticator.oidc/src/test/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticatorTest.java

```diff
@@ -118,6 +122,9 @@
         "org.wso2.carbon.identity.application.authentication.framework.exception.AuthenticationFailedException"})
 public class OpenIDConnectAuthenticatorTest extends PowerMockTestCase {
 
+    public static final String OIDC_PARAM_MAP_STRING = "oidc:param.map";
+    public static final String HTTPS_LOCALHOST_9443 = "https://localhost:9443";
+    public static final String COMMA_SEPARATOR = ",";
```

Comment:
can make this private

Reply:
Fixed df6a9e1d41e3263359fdf606457051fe65194a63